### PR TITLE
Polish threads misalignments and UI diversion

### PR DESCRIPTION
**Pull request by gsouquet**
Fixes https://github.com/vector-im/element-web/issues/19772
Fixes https://github.com/vector-im/element-web/issues/19710
Fixes https://github.com/vector-im/element-web/issues/19629
Fixes https://github.com/vector-im/element-web/issues/19711

- Fix reply padding in thread view
- fix hover and attachment alignment in threads
- fix thread panel preview
- Fix timestamp on hover in thread panel

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Polish threads misalignments and UI diversion ([\#7209](https://github.com/matrix-org/matrix-react-sdk/pull/7209)). Fixes vector-im/element-web#19772, vector-im/element-web#19710 vector-im/element-web#19629 and vector-im/element-web#19711.

Preview: https://61a4a94121f2820e6d18fe48--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by SimonBrandner — CHANGES_REQUESTED** [review]
Screenshot_20211126_162146

This breaks the shape of the hovered tile border, I don't think there is a way to make this work with `box-shadow`

**Comment by SimonBrandner** [comment]
(the tile border shape was deliberately changed to what it looks like now in https://github.com/matrix-org/matrix-react-sdk/pull/6525, fyi)

**Comment by gsouquet** [comment]
I tweaked the `box-shadow` a bit and it now looks just like in production. I prefer that solution rather than using border as we don't have to reset the `padding` on hover

Screen Shot 2021-11-29 at 09 23 55

**Comment by SimonBrandner** [comment]
> I tweaked the `box-shadow` a bit and it now looks just like in production. I prefer that solution rather than using border as we don't have to reset the `padding` on hover
> 
> Screen Shot 2021-11-29 at 09 23 55

Looks awesome! It also makes more sense to me but I didn't think this was possible... I think there are still a few places in event tile that use border, could you change those too as well for consistency?

**Comment by SimonBrandner** [comment]
Screenshot_20211129_103149

Also, this is quite an interesting bug :D

**Comment by gsouquet** [comment]
@SimonBrandner Haven't found how to reproduce that bug. But i've changed all `border-left` to `box-shadow` so that should help a fair bit in this scenario

**Comment by SimonBrandner** [comment]
> @SimonBrandner Haven't found how to reproduce that bug. But i've changed all `border-left` to `box-shadow` so that should help a fair bit in this scenario

It happened when I hovered a selected event

File: src/components/views/rooms/EventTile.tsx [hunk 1]
```diff
@@ -1362,22 +1362,9 @@ export default class EventTile extends React.Component<IProps, IState> {
...
-                                showUrlPreview={this.props.showUrlPreview}
-                                permalinkCreator={this.props.permalinkCreator}
-                                onHeightChanged={this.props.onHeightChanged}
-                                callEventGrouper={this.props.callEventGrouper}
-                                tileShape={this.props.tileShape}
-                                getRelationsForEvent={this.props.getRelationsForEvent}
-                            />
-                            { keyRequestInfo }
+                            <div className="mx_EventTile_body">
+                                { MessagePreviewStore.instance.generatePreviewForEvent(this.props.mxEvent) }
```
Comment (SimonBrandner) [inline src/components/views/rooms/EventTile.tsx:1366]: I am a little confused by this change... Why are we doing this? It seems like the `callEventGrouper` isn't getting to the event?

Comment (gsouquet) [reply src/components/views/rooms/EventTile.tsx:1366]: There won't be any call events in a thread. This is to tackle this https://github.com/vector-im/element-web/issues/19710#issuecomment-975730760

**Review by t3chguy — APPROVED** [review]

File: src/components/views/rooms/EventTile.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/components/views/rooms/EventTile.tsx:1366]: Oh, oops, didn't notice the full context of the code :facepalm:

**Review by SimonBrandner — APPROVED** [review]
It seems to work! And I trust @t3chguy's code review :D

**Pull request merged**